src/LinearSolvers/LinearSolvers.jl

```diff
@@ -160,7 +160,7 @@ function ldiv!(solver::BICGSTAB,
     y[:], n_iters, status = bicgstab(J, x, solver.precond, y; maxiter=solver.maxiter,
                                      verbose=solver.verbose, tol=solver.tol)
     if status != Converged
-        @warn("BICGSTAB failed to converge. Final status is $(status)")
+        error("BICGSTAB failed to converge. Final status is $(status)")
```

Comment:
`error`??? You are rough :-) . Maybe one wants to continue doing something no?

Reply:
Right ... What do you think it's better? Issuing a warning with `@warn` or log something with `println`?

Reply:
I found `@warn` appropriate. I'd say `@info` or `@warn`. I think I'd prefer `@warn` because one may want to turn off normal output on larger systems, but may still want to be warned about this.